care/emr/migrations/0010_facilitylocation_facilitylocationorganization.py

```
@@ -0,0 +1,71 @@
+                ('description', models.CharField(max_length=255)),
+                ('mode', models.CharField(max_length=255)),
+                ('location_type', models.CharField(max_length=255)),
+                ('form', models.CharField(max_length=255)),
+                ('facility_organization_cache', django.contrib.postgres.fields.ArrayField(base_field=models.IntegerField(), default=list, size=None)),
+                ('has_children', models.BooleanField(default=False)),
+                ('level_cache', models.IntegerField(default=0)),
+                ('parent_cache', django.contrib.postgres.fields.ArrayField(base_field=models.IntegerField(), default=list, size=None)),
+                ('metadata', models.JSONField(default=dict)),
+                ('cached_parent_json', models.JSONField(default=dict)),
+                ('created_by', models.ForeignKey(blank=True, default=None, null=True, on_delete=django.db.models.deletion.SET_NULL, related_name='%(app_label)s_%(class)s_created_by', to=settings.AUTH_USER_MODEL)),
+                ('facility', models.ForeignKey(on_delete=django.db.models.deletion.PROTECT, to='facility.facility')),
```

Ruff (0.8.2)

Single quotes found but double quotes preferred

Replace single quotes with double quotes

(Q000)